l5kit/l5kit/rasterization/render_context.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+
+class RenderContext:
+    def __init__(
+        self, raster_size_px: np.ndarray, pixel_size_m: np.ndarray, center_in_raster_ratio: np.ndarray,
+    ) -> None:
+        """
+        This class stores render context information (raster size, pixel size, raster center / principle point) and
+        it computes a transformation matrix (raster_from_local) to transform a local pose to the raster center
```

Comment:
'to the raster center' sounds weird - this matrix defines a constant offset to the raster's center? Is it really a transformation matrix then, does it transform every point to this center?

Reply:
I guess it's just means that the local centre ends up in the raster centre. Let me rephrase it a bit :)